### PR TITLE
Add OISD-nsfw blocklist

### PR DESCRIPTION
**Pull request by smith558**
Could [OISD NSFW blocklist](https://oisd.nl/downloads) be added as an option to the blocklists available? There are just **soooo many** sites missing in the default "porn" list currently used in NextDNS that it's basically useless for any porn/adult content control.

We would really like to purchase NextDNS Business/ Education but with the adult content protection being basically non-existent we will have to go search for a different provider. Just a simple search for the highest traffic adult sites and testing them on your filter shows dozens passing through. The OIDS list is the most extensive and I found it to contain most sites the default NextDNS list isn't catching.

### CODE REVIEW

**Comment by smith558** [comment]
@romaincointepas please kindly let me know if this is doable

**Comment by Mireli5656** [comment]
Nextdns also has a porn blocker in the parental controls section, there's no need to add this to the blocklist.
